src/lightning/app/runners/cloud.py

```diff
@@ -539,6 +550,23 @@ def _resolve_cloudspace_name(
             cloudspace_name = random_name
         return cloudspace_name
 
+    def _resolve_run_name(
+        self,
+        name: str,
+        existing_instances: List[Externalv1LightningappInstance],
+    ) -> str:
+        """If there are existing instances with the same name - choose a randomised name."""
+        if len(existing_instances) > 0:
+            letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
```

Comment:
You can use python built-in: https://docs.python.org/3/library/string.html

Reply:
Nice! Done 😃